readme.md

```diff
@@ -26,7 +40,7 @@ Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
 ```text
-
+In order to send a form via ajax using jquery we store the values of the in inputs of our form into and object that we send via a post ajax call in json format to the server. JQuery is just used for the ajax call.
```

Comment:
:+1: 

It would be nice to see something about what happens after the ajax call.